linux_os/guide/services/ntp/chronyd_run_as_chrony_user/ansible/shared.yml

```diff
@@ -0,0 +1,30 @@
+# platform = multi_platform_fedora,Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+
+- name: "detect if file is not empty or missing"
```

Comment:
Please, capitalize the name of the tasks.

Reply:
Also, be more specific of what it is doing. If this task is part of a playbook and I looked at the log, I wouldn't know what this is doing.